### DISCUSSION TITLE
build error from other project

### DISCUSSION THREAD

**Discussion by HaSaKiYasuooo** [jreleaser/jreleaser#1560] [Q&A]
In accordance with the https://github.com/gotson/komga/blob/master/DEVELOPING.md#docker and https://github.com/gotson/komga I build docker image file, And using the jreleaser inside, the following issues arise.
![image](https://github.com/jreleaser/jreleaser/assets/87591236/701e572f-48fb-4874-b687-6012f562b946)

**Comment by aalmiray** [comment]
The error message is telling you that a value for a Github token is require. The preferred way to supply such token is via an environment variable named `JRELEASER_GITHUB_TOKEN`.
